packages/scene-graph-mediator/runtime/src/property_converter/Pixi.ts

```diff
@@ -43,44 +43,22 @@ export const Pixi: PropertyConverter.Interface = {
   },
 
   fixCoordinate: (
-    target: any,
+    schema: SchemaJson,
     convertedObject: ConvertedObject,
-    node: Node,
-    parentNode?: Node
+    node: Node
   ): void => {
-    const transform = node.transform;
-
-    if (parentNode) {
-      const scale = transform.scale || { x: 1, y: 1 };
-
-      convertedObject.position.x += (
-        (parentNode.transform.width  || 0) - (transform.width  || 0) * scale.x
-      ) * transform.anchor.x;
-      convertedObject.position.y += (
-        (parentNode.transform.height || 0) - (transform.height || 0) * scale.y
-      ) * transform.anchor.y;
-    }
-
-    if (target.anchor) {
-      const size = {
-        width: target.width,
-        height: target.height
+    if (!node.transform.parent) {
+      const sceneBasePoint = {
```

Comment:
property name should be x/y for variable named `sceneBasePoint`.

Reply:
fix 'sceneBaseProint' property name 'width/height' to 'x/y'. https://github.com/drecom/scene-graph/pull/36/commits/829680a3545f75a18add7d3ce481cd3df5f51129